client/app/recommendationList.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { RecommendationListEntry } from './recommendationListEntry.jsx';
+
+const RecommendationList = ({ recs }) => {
+  const recs1 = recs.slice(0, 4);
+  const recs2 = recs.slice(4, 8);
+  const recs3 = recs.slice(8, 12);
+  return (
+    <div>
+      <div>{recs1.map(item => <RecommendationListEntry rec={item} />)}</div>
```

Comment:
since you are already returning your entries in a div, i dont think you need to enclose them as divs out here